go/util/jwt/jwt.go

```diff
@@ -106,6 +101,32 @@ func (c PermissionScopes) Has(scope PermissionScope) bool {
 	return false
 }
 
+func (c *PermissionScopes) UnmarshalCSV(val string) error {
```

Comment:
Suggestion: have the CSV unmarshaler in its own package following the same signature as the `json.Unmarshal(data []byte, v any) error`. Makes the code more standard and this function reusable. From an encapsulation perspective CSV unmarshalling logic is now overlapping with JWT code, having it in its own package would improve this as well.

Reply:
So, unfortunately, marshaling is undiscovered territory, each language makes it own way`. Technically `UnmarshalCSV` is following what what others do, eg `UnmarshalJSON` and `UnmarshalYAML`.
And, CSV parser from stdlib does not support Unmarshaling anyways.